2021-10-26 17:03:52,288 INFO    MainThread:4328 [wandb_setup.py:_flush():71] setting env: {'run_group': 'dev', 'project': 'exercise_14'}
2021-10-26 17:03:52,289 INFO    MainThread:4328 [wandb_setup.py:_flush():71] setting login settings: {}
2021-10-26 17:03:52,289 INFO    MainThread:4328 [wandb_init.py:_log_setup():357] Logging user logs to /home/default/github/genre_classification/starter/download/wandb/run-20211026_170352-2qafmnt3/logs/debug.log
2021-10-26 17:03:52,289 INFO    MainThread:4328 [wandb_init.py:_log_setup():358] Logging internal logs to /home/default/github/genre_classification/starter/download/wandb/run-20211026_170352-2qafmnt3/logs/debug-internal.log
2021-10-26 17:03:52,289 INFO    MainThread:4328 [wandb_init.py:init():390] calling init triggers
2021-10-26 17:03:52,289 INFO    MainThread:4328 [wandb_init.py:init():395] wandb.init called with sweep_config: {}
config: {}
2021-10-26 17:03:52,290 INFO    MainThread:4328 [wandb_init.py:init():435] starting backend
2021-10-26 17:03:52,290 INFO    MainThread:4328 [backend.py:_multiprocessing_setup():94] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-10-26 17:03:52,313 INFO    MainThread:4328 [backend.py:ensure_launched():198] starting backend process...
2021-10-26 17:03:52,321 INFO    MainThread:4328 [backend.py:ensure_launched():203] started backend process with pid: 4341
2021-10-26 17:03:52,322 INFO    MainThread:4328 [wandb_init.py:init():444] backend started and connected
2021-10-26 17:03:52,324 INFO    MainThread:4328 [wandb_init.py:init():503] updated telemetry
2021-10-26 17:03:52,340 INFO    MainThread:4328 [wandb_init.py:init():533] communicating current version
2021-10-26 17:03:52,956 INFO    MainThread:4328 [wandb_init.py:init():538] got version response 
2021-10-26 17:03:52,956 INFO    MainThread:4328 [wandb_init.py:init():548] communicating run to backend with 30 second timeout
2021-10-26 17:03:53,380 INFO    MainThread:4328 [wandb_init.py:init():576] starting run threads in backend
2021-10-26 17:03:56,212 INFO    MainThread:4328 [wandb_run.py:_console_start():1693] atexit reg
2021-10-26 17:03:56,213 INFO    MainThread:4328 [wandb_run.py:_redirect():1567] redirect: SettingsConsole.REDIRECT
2021-10-26 17:03:56,213 INFO    MainThread:4328 [wandb_run.py:_redirect():1572] Redirecting console.
2021-10-26 17:03:56,215 INFO    MainThread:4328 [wandb_run.py:_redirect():1628] Redirects installed.
2021-10-26 17:03:56,215 INFO    MainThread:4328 [wandb_init.py:init():603] run started, returning control to user process
2021-10-26 17:04:00,272 INFO    MainThread:4328 [wandb_run.py:finish():1347] finishing run default141/exercise_14/2qafmnt3
2021-10-26 17:04:00,272 INFO    MainThread:4328 [wandb_run.py:_atexit_cleanup():1663] got exitcode: 0
2021-10-26 17:04:00,272 INFO    MainThread:4328 [wandb_run.py:_restore():1635] restore
2021-10-26 17:04:03,225 INFO    MainThread:4328 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1234
  total_bytes: 1234
}

2021-10-26 17:04:03,639 INFO    MainThread:4328 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1234
  total_bytes: 1234
}

2021-10-26 17:04:04,400 INFO    MainThread:4328 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1234
  total_bytes: 3840
}

2021-10-26 17:04:04,501 INFO    MainThread:4328 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1234
  total_bytes: 3866
}

2021-10-26 17:04:04,602 INFO    MainThread:4328 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1234
  total_bytes: 3866
}

2021-10-26 17:04:04,705 INFO    MainThread:4328 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1234
  total_bytes: 3866
}

2021-10-26 17:04:04,806 INFO    MainThread:4328 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1234
  total_bytes: 3866
}

2021-10-26 17:04:04,908 INFO    MainThread:4328 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 3866
  total_bytes: 3866
}

2021-10-26 17:04:05,010 INFO    MainThread:4328 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 3866
  total_bytes: 3866
}

2021-10-26 17:04:05,111 INFO    MainThread:4328 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 3866
  total_bytes: 3866
}

2021-10-26 17:04:05,213 INFO    MainThread:4328 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 3866
  total_bytes: 3866
}

2021-10-26 17:04:05,314 INFO    MainThread:4328 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 3866
  total_bytes: 3866
}

2021-10-26 17:04:05,416 INFO    MainThread:4328 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 3866
  total_bytes: 3866
}

2021-10-26 17:04:05,764 INFO    MainThread:4328 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 3866
  total_bytes: 3866
}

2021-10-26 17:04:06,222 INFO    MainThread:4328 [wandb_run.py:_wait_for_finish():1793] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 3866
  total_bytes: 3866
}
local_info {
}

2021-10-26 17:04:07,295 INFO    MainThread:4328 [wandb_run.py:_append_files():2061] logging synced files
